doc/manual/source/threat-model.rst

```diff
@@ -55,11 +51,11 @@ An adversary with network access on-path between Routinator and a repository cou
 * Block access to the repository or degrade throughput of access to repository data
 * Downgrade the transport used for repository access from RRDP (which provides confidentiality and integrity at the transport level) to rsync (which does not). This behaviour was decided by the community. After such a downgrade, the attacker can: 
 
-  * Inject objects, withhold objects and manipulate objects, leading to rejection of otherwise valid repository data.
+  * Inject objects, withhold objects and manipulate objects at the transport level, leading to rejection by Routinator of otherwise valid repository data  
```

Comment:
```suggestion
  * Inject objects, withhold objects and manipulate objects at the transport level, leading to rejection by Routinator of otherwise valid repository data
```